docs/ide4c/vscode/launch.json

```diff
@@ -0,0 +1,66 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "intltest Launch", 
+            "type": "cppdbg",
+            "request": "launch",
+            "targetArchitecture": "x64",
+            "program": "${workspaceFolder}/test/intltest/intltest",
+            "environment": [
+                {
+                    "name": "LD_LIBRARY_PATH", // Use "DYLD_LIBRARY_PATH" for mac osx.
+                    "value": "${workspaceRoot}/lib:${workspaceRoot}/stubdata:${workspaceRoot}/tools/ctestfw"
+                },
+                {
+                    "name": "ICU_DATA",
+                    "value": "${workspaceRoot}/data/out"
+                }
+            ],
+            "args": [ // Here you add wich test cases you want to run, leaving it empty will run all the test cases.
+             "rbbi/RBBITest" // Just for example.
+            ],
+            "stopAtEntry": true, // If you want to add a break point in the begining of the program (i.e. test cases).
```

Comment:
Why default to true instead of false?  It is kind-of confusing to step from the main function down into your test case.  I usually leave a breakpoint in my test case and let the code run until it gets there.

Reply:
because the debugging can start even there is no break-points have been set. However, I agree with you. I will make the default  value false.